src/utils.ts

```diff
@@ -21,6 +21,7 @@ import * as os from 'os'
 import * as child_process from 'child_process'
 import path from 'path'
 import { VSCodeLaunchConfigArgs } from './classes/vscode-launch'
+import * as JSONC from 'jsonc-parser'
```

Comment:
Looks like you're just using `parse` from from `json-parser`, why `import *` from it?